### PR TITLE
Updates GitHub provider resource descriptions

### PR DESCRIPTION
**Pull request by scottford-io**
Updates resource descriptions for GitHub

Signed-off-by: Scott Ford <scott@scottford.io>

### CODE REVIEW

**Comment by tas50** [comment]
https://github.com/mondoohq/cnquery/pull/658 needs to get merged first. There was typos in the previous PR that are causing failures as these make it to the docs site.

File: resources/packs/github/github.lr [hunk 1]
```diff
@@ -381,7 +399,7 @@ private github.commit @defaults("sha") {
 
 // GitHub Repository Pull Request 
 private github.mergeRequest @defaults("id state") {
-  // Pull request ID
+  // Pull request id
```
Comment (misterpantz) [inline resources/packs/github/github.lr]: Hey @scottford-io why the lowercase id here and below?

**Review by misterpantz — COMMENTED** [review]
Thank you for this!!!

File: resources/packs/github/github.lr (hunk 1 above)
Comment (chris-rock) [reply resources/packs/github/github.lr]: I also think ID is upper case

Comment (scottford-io) [reply resources/packs/github/github.lr]: agreed!

Comment (scottford-io) [reply resources/packs/github/github.lr]: fixed!

**Review by tas50 — APPROVED** [review]

**Pull request merged**